src/utils/ack-timeout-registry.js

```diff
@@ -32,12 +32,16 @@ EventEmitter( AckTimeoutRegistry.prototype );
  * @public
  * @returns {void}
  */
-AckTimeoutRegistry.prototype.add = function( name ) {
-	if( this._register[ name ] ) {
-		throw new Error( 'Timeout ' + name + ' is already registered' );
+AckTimeoutRegistry.prototype.add = function( name, action ) {
+	var uniqueName = action ? action + name : name;
+	
+	if( this._register[ uniqueName ] ) {
+		this.clear( {
+			data: [ action, name ]
```

Comment:
its an ACK reply so action is ACK and the real action is data[0]

R|A|S|recordName

action: A
data: [ S, recordName ]